packages/hardhat-core/src/builtin-tasks/test.ts

```diff
@@ -1,4 +1,5 @@
 import chalk from "chalk";
+import type { MochaOptions } from "mocha";
```

Comment:
Personally I always like type-exclusive imports to be first, but that's subjective and certainly not a blocker for merging since I don't think it's a standard we adhere to across the codebase.

Reply:
That's a nice idea. Can we add a linter rule for it?